TeltonikaMonitor/package.json

```diff
@@ -5,20 +5,19 @@
     "prettier": "@camstreamer/prettier-config",
     "main": "src/main.ts",
     "scripts": {
-        "in:prod": "npm install --omit dev",
-        "in:all": "npm install",
-        "clean": "rimraf dist/* html/*",
-        "try": "ts-node src/main.ts",
-        "start": "cd react && npm run start && cd ..",
-        "pretty": "prettier --write \"./{src,react/src,react/src/components}/*.{ts,tsx}\"",
         "lint": "eslint \"./{src,react/src}/**/*.ts\"",
         "lint:fix": "eslint \"src/**/*.ts\" --fix",
+        "pretty": "prettier --write \"./{src,react/src,react/src/*}/*.{ts,tsx}\"",
+        "pretty:check": "prettier --check \"./{src,react/src,react/src/*}/*.{ts,tsx}\"",
+        "try": "PERSISTENT_DATA_PATH=\"./localdata/\" ts-node src/main.ts",
+        "start": "cd react && npm run start",
+        "clean": "rimraf dist/* html/*",
+        "in:ui": "cd react && npm install",
         "build:BE": "tsc",
-        "build:FE": "cd react && npm run build && cd ..",
+        "build:FE": "cd react && npm run build",
         "build": "npm-run-all build:BE build:FE",
-        "prepare-zip": "npm-run-all build in:prod",
         "zip:package": "node node_modules/camstreamerlib/CreatePackage -i -e=react,testing",
-        "zip": "npm-run-all clean prepare-zip zip:package in:all"
+        "zip": "npm install; npm-run-all clean in:ui build zip:package"
```

Comment:
kolik casu zabere npm install? nedame to pryc? precejenom dat npm install a pak npm run zip uz neni takovy problem a je to standard..pak kdyz vyvijisi a pridas modul tak ho mas take rovnou nainstalovany..

Reply:
!